### ISSUE TITLE
Error with the printf arguments

### ISSUE THREAD

**Issue by ncou** [ramsey/coding-standard#17]
Hi,

Nice piece of code. I think there is an error in the xmllint bash during the printf function. The "specifier" number is not at the right place. You have used "%$1s" instead of "%1$s" so there is an error.

https://github.com/ramsey/coding-standard/blob/main/bin/xmllint#L54

I think you should use `'%1$sI found the following issues with %2$s.%1$sYou may fix these issues automatically with:%1$s%1$s  composer lint:fix%1$s%1$s%3$s%1$s',`

Hope it's helping.

Keepup the good work !

**Comment by ramsey** [comment]
What is the error message?

**Comment by ramsey** [comment]
I just looked at the docs, and I think you're correct. I'm curious why I've never noticed a problem. 🤔

Feel free to submit a pull request. I can help with this, if you like. Otherwise, I'll try to get it fixed in the next week or so.

Thanks!

**Comment by ncou** [comment]
Yes it's strange you never see the problem. I have made a quick test in php7.4 and php8 and in php7.4 it display a warning, and in php 8 it throw a fatal error.

`Warning:  printf(): Argument number must be greater than zero in...`
`Fatal error:  Uncaught ValueError: Argument number must be greater than zero in...` 

I will submit a PR in a few hours.

**Comment by ramsey** [comment]
Thanks!

**Comment by ncou** [comment]
I have created the PR. Tell me if it's ok.

By the way the hook for the pre-commit are a bit strange for the `descriptionEndMark` value. It's empty so it mean there should not have a ponctuation character at the end of the description. It's not really clear the character ")" is considered as a punctuation. I got an error on the message "fix: blablabla in the function printf()" but not a lot of information for the error. But removing the trailing "()" allowed me to commit the message. Hope i am clear english is not my mother tongue.
